src/plugins/commander.go

```diff
@@ -88,8 +88,23 @@ func (c *Commander) Process(msg *core.Message) {
 
 func (c *Commander) agentBackoff(agentConfig *proto.AgentConfig) {
 	log.Debugf("update commander client configuration to %+v", agentConfig)
-	backOffSettings := sdk.ConvertBackOffSettings(agentConfig.Details.Server.GetBackoff())
+
+	if agentConfig.GetDetails() == nil || agentConfig.GetDetails().GetServer() == nil || agentConfig.GetDetails().GetServer().GetBackoff() == nil {
+		log.Debug("not updating commander client configuration as new Agent backoff settings is nil")
```

Comment:
Once server is out of planned outage mode, where should we reset agent backoff settings back? We do not want to have settings that server provided when it went in outage mode.

Reply:
The agent is stateless, so it's up to the server to tell us what backoff settings they want. If you change the backoff settings and then want to update them again back to the default backoff setting, it is the server that need to send us the default backoff settings again in the agent config command.